man/setDT.Rd

```diff
@@ -58,6 +59,23 @@ setDT(X, key="a")[]
 X = list(a=1:5, a=6:10)
 setDT(X, check.names=TRUE)[]
 
+# FAQ: Dealing with data.table loaded from RDS or RData files
+# Example to demonstrate the usage of setDT after loading from RDS
+# Step 1: Define file path
+rds_file = tempfile("my_data.rds_")
+# Step 2: Create and save data.table if the file does not exist
+if (!file.exists(rds_file)) {
```

Comment:
do we need this `if()`? Why would the file exist? This feels like needless clutter